estate/views/estate_property_offer_view.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="estate_property_offer_view_form" model="ir.ui.view">
+        <field name="name">estate.property.offer.form</field>
+        <field name="model">estate.property.offer</field>
+        <field name="arch" type="xml">
+            <form string="Property Offer">
+                <header>
+                </header>
+                <sheet>
+                    <div class="oe_title">
+                        <div class="oe_edit_only">
+                            <label for="price"/>
+                        </div>
+                        <h1>
+                            <field name="price" string="Price"/>
+                        </h1>
+                    </div>
+                    <div>
+                        <group>
+                            <field name="status"/>
+                            <field name="partner_id"/>
+                            <field name="property_id"/>
+                            <field name="validity"/>
+                            <field name="date_deadline"/>
+                        </group>
+                    </div>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="estate_property_offer_view_list" model="ir.ui.view">
+        <field name="name">estate.property.offer.view.list</field>
+        <field name="model">estate.property.offer</field>
+        <field name="arch" type="xml">
+            <list string="Property Offer List" editable="bottom"
+                  decoration-success="status == 'accepted'"
+                  decoration-danger="status == 'refused'">
+                <field name="price"/>
+                <field name="validity"/>
+                <field name="date_deadline"/>
+                <field name="partner_id"/>
+                <field name="status" column_invisible="1"/>
+                <button name="action_accept_offer" string="Confirm" type="object" icon="fa-check"
+                        invisible="bool(status)"/>
+                <button name="action_refuse_offer" string="Refuse" type="object" icon="fa-times"
+                        invisible="bool(status)"/>
+            </list>
+        </field>
+    </record>
+
+    <record id="estate_property_offer_model_action" model="ir.actions.act_window">
```

Comment:
```suggestion
    <record id="estate_property_offer_action" model="ir.actions.act_window">
```

Convention for naming actions is `{model_name}_action`
Convention for naming view is `{model_name}_view{_type}`